tools/minify.py

```diff
@@ -0,0 +1,34 @@
+# </DOCKER_BUILD>
+# minify collected javascript files
+# ---------------------------------
+# RUN python tools/minify.py
+
+"""
+Minify collected javascript files
+
+This would ideally be moved into either the frontend building pipeline (gulp
+et al.) or integrated into the building process (collectstatic or another
+management command).
+"""
+import os
+import gzip
+
+import rjsmin
+
+
+def minify(staticfiles_path):
+    for root, dirs, files in os.walk(staticfiles_path):
+        for file in files:
+            if not file.endswith('.js'):
+                continue
+            file = os.path.join(root, file)
+            with open(file, 'r+') as fh:
+                minified = rjsmin.jsmin(fh.read())
```

Comment:
e.g. i don't want my script that i carefully prepared and minified already to be mangled by something else on top of it